### DISCUSSION TITLE
Information from Professor Deza

### DISCUSSION THREAD

**Discussion by kbuzard** [ecn310/course-project-zipcentercrime#4] [General]
@ecn310/zipcentercrime Below is the information that Professor Deza shared with me to help you get started. For your first paper summary, start with the *Local access to mental healthcare and crime* paper.

### Research Question
Does having a substance abuse center in your zip code affects which types of crime or crime at all, as measured by 911 calls in Detroit?

### Data
Clean up a subset of the data by imputing zip codes (data has intersection, but not zip code), then merge with substance use centers at the zip code level.

[Here](https://www.dropbox.com/scl/fi/mvlni30fz74qx4fclofmc/calls_final.csv?rlkey=drs9rkqlgyo9i8gsf9823prof&dl=0) is the 911 call data.
- It's about 1.4 GB, so can't go on Github
- The variable incident_address has the intersection. There are a lot of missing zip codes, so it would need to impute those zip codes first. 

### Literature
- Local access to mental healthcare and crime, https://www.sciencedirect.com/science/article/pii/S0094119021000929
- https://www.montana.edu/swensen/documents/MentalHealth_SSI_forCFS_final.pdf
- https://www.nber.org/papers/w22610

**Comment by regaudre** [comment]
## Agenda for meeting with Prof. Deza
Research Question: "Does having a substance abuse center in your zip code affects which types of crime or crime at all, as measured by 911 calls in Detroit?"

#### General Questions about the topic and research
- Why are you interested in this research question?
- Is there any general background information we might need when studying this topic or Economics of Crime in general?
- Did you have an expectation of what the findings might look like, or any expectations where this might lead?
- What are you hoping to get out of us studying this question?
#### Data
- Where did you get the "calls_final" data from? (We need the origin information)
- Do you have compiled data on substance abuse center locations? (We also need this datas origin)
- The calls_final data set includes a lot of observations outside of Detroit city limits. Do you have a reccommendation for how far the geographical area that we are looking at should exten? (we are currently thinking Detroit, Highland Park, and Hamtramck)
- We're looking at the crime and call data based on zipcode. The calls final data also contains data from 2016 - 2022. Should we isolate the data into a shorter time frame and compare across zipcodes?
